src/lib/components/WellCompletions/components/Settings/__snapshots__/HideZeroCompletionsSwitch.test.tsx.snap

```diff
@@ -2,26 +2,26 @@
 
 exports[`test hide zero completions switch snapshot test 1`] = `
 <label
-  className="sc-dUrnRO eauNHr"
```

Comment:
I am wondering why this gets updated each time. It might be to do with how styled component generates hashed className. I will try to fix this in another PR